pymeasure/instruments/keithley/keithley_2600.py

```diff
@@ -82,7 +82,7 @@ def ask(self, cmd):
     def write(self, cmd):
         self.instrument.write(f'smu{self.channel}.{cmd}')
 
-    def values(self, cmd, **kwargs):
+    def values(self, cmd):
```

Comment:
Why this change?

If we keep it, please change the docstring accordingly (currently it states, that it hands over the keyword arguments).